script/Arcueid/Day 6/06_00_ARC06_1.txt

```diff
@@ -424,7 +423,7 @@ You cried out a little earlier, but if I were to put in a word with Akiha-sama..
 [sha:6461078e709e48d361f82e3afbb92e2133b48de4]{
 -- Page 33, Offset 14004.
 -- それより秋葉の方が怖い。先延ばしにしたら、よけいに怒られそうだ」
-Besides, Akiha is a much more pressing concern. If I keep making her wait, she'll just get even angrier."
+Besides, Akiha is way more pressing. If I keep making her wait, she'll just get even angrier."
```

Comment:
Might prefer 
> Besides, I'm much more worried about Akiha than my leg.

Since _Akiha_ isn't pressing so much as _attending to Akiha_ is pressing